### PR TITLE
feat: add preact-flicking package

### PR DESCRIPTION
**Pull request by daybrush**
## Issue

#234

### CODE REVIEW

**Review by mixed — APPROVED** [review]
Welcome to Flicking for @preactjs

File: packages/preact-flicking/src/preact-flicking/Flicking.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import Flicking from "@egjs/react-flicking";
+
+export default Flicking;
```
Comment (WoodNeck) [inline packages/preact-flicking/src/preact-flicking/Flicking.js:3]: I think there could be a reason to make this file in `js` format, can I ask why?

**Review by WoodNeck — APPROVED** [review]
LGTM

File: packages/preact-flicking/src/preact-flicking/Flicking.js (hunk 1 above)
Comment (daybrush) [reply packages/preact-flicking/src/preact-flicking/Flicking.js:3]: @WoodNeck 

One of the ways to convert from react to preact is to remove the react type.
So, I created Flicking.js and Flicking.d.ts.  React was removed from Flicking.d.ts and changed to preact.

**Pull request merged**